travis/Dockerfile-travis-web

```diff
@@ -13,5 +13,6 @@ WORKDIR /src
 COPY . /src
 
 RUN chown -R mitodl:mitodl /src
+RUN mkdir /var/media && chown -R mitodl:mitodl /var/media
```

Comment:
Also the container built with this dockerfile only persists for the duration of the tests, do we need this directory at all?